cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -2,15 +2,216 @@ package qcow2ova
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/manifoldco/promptui"
+	"github.com/ppc64le-cloud/pvsadm/cmd/image/qcow2ova/ova"
+	"github.com/ppc64le-cloud/pvsadm/cmd/image/qcow2ova/prep"
+	"github.com/ppc64le-cloud/pvsadm/cmd/image/qcow2ova/validate"
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
 	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
 )
 
 var Cmd = &cobra.Command{
 	Use:   "qcow2ova",
 	Short: "Convert the qcow2 image to ova format",
-	Long:  `Convert the qcow2 image to ova format`,
+	Long: `Convert the qcow2 image to ova format
+
+Examples:
+
+  # Downloads the coreos image from remote site and converts into ova type with name rhcos-461.ova.gz
```

Comment:
I was wondering should we stick to coreos or change it to rhcos ? I have no preference

Reply:
me either, I think we can keep this coreos and can be helpful for fedora coreos as well in the future, wdys?

Reply:
yeah.. make sense. let's stick to coreos.